hw/top_earlgrey/dv/cov/chip_cover.cfg

```diff
@@ -11,6 +11,15 @@ begin tgl(portsonly)
   +module padring
   +module rv_core_ibex
   +moduletree top_earlgrey 2
+
+  // [LOW_RISK] These are covered in FPV. These are partly covered in DV as well, but it is not
+  // feasible to get a full 100% coverage on all of the bits. The rationale for excluding these from
+  // coverage collection entirely is documented in lowrisc/opentitan#15427.
+  -node tb.dut.top_earlgrey.u_rv_core_ibex.u_core.crash_dump_o
```

Comment:
@tjaychen, fyi, this is an example of compile time coverage exclusion for your PR #15723.

Reply:
thanks for the example and reminder! i will hopefully wrap up that one sometime this week...